celery/concurrency/asynpool.py

```diff
@@ -508,7 +509,8 @@ def register_with_event_loop(self, hub):
         self.handle_result_event = self._result_handler.handle_event
         self._create_timelimit_handlers(hub)
         self._create_process_handlers(hub)
-        self._create_write_handlers(hub)
+        self._create_write_handlers(hub,
```

Comment:
please add test and docs for this change

Reply:
there are no tests for asynpool 
I ran all tests on debug and it got there only in 2 tests that were tested worker controller
what do you suggest?

Reply:
this is why `serializer_registry._encoders['pickle'][-1](obj, protocol=protocol)` passed
it fails when trying running it locally because it doesn't have protocol param 
I'm fixing it, but `asynpool` isn't being testing as part of the uts